### DISCUSSION TITLE
Tables not appearing when fetching from postgresql

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#4739] [Help] [answered]
I am connected to a locally running instance of postgresql. My connection info is correct, the fetch tables says it completes correctly, but the tables do not appear in budibase. Creating a table works properly.

**Comment by mjashanks** [comment, marked as answer]
Hey @gondilon - are you maybe connecting to the wrong schema? It defaults to public:

![image](https://user-images.githubusercontent.com/3524181/156535248-db063ae3-e135-4083-8e2b-d2a190b39336.png)

Another possible issue is that your tables do not have primary keys (though Budubase usually warns you when this happens)
